frontend/src/api.js

```diff
@@ -5,11 +5,18 @@ import _isEmpty from "lodash/isEmpty";
 import config from "./config";
 import { base64ToBlob } from "./helper";
 import strings from "./localizeStrings";
+import { store } from "./store";
 
 const devMode = config.envMode === "development";
 const API_VERSION = "1.0";
 const instance = axios.create();
 
+const isAuthProxyEnabled = window?.injectedEnv?.REACT_APP_AUTHPROXY_ENABLED === "true" || config.authProxy.enabled;
+const authProxySignoutUri = (window?.injectedEnv?.REACT_APP_AUTHPROXY_URL || config.authProxy.url).replace(
```

Comment:
@SamuelPull this feels a little strange. I would probably prefer separate variable for signout or and appendix to the original URL, but we can keep it like this.

Reply:
Yeah, it would have been better to use SOME_URL and append to it, but that's a breaking change now. Let's keep it like this. Not great, not terrible.